src/components/progress/progressRing.svelte

```diff
@@ -84,7 +84,7 @@
       animation-duration: var(--spin-rate);
       animation-name: spin;
       animation-iteration-count: infinite;
-      animation-timing-function: linear;
+      animation-timing-function: var(--leo-progressring-easing, linear);
```

Comment:
shouldn't this new easing be applied all the time if it's the DS progressRing style? When would we want it to be different?